### PR TITLE
[Edit] Add notes about overloading for C++ Classes

### LINKED ISSUE #3398: [Edit] C++ Add Notes: C++ Classes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to make an edit for C++ Classes. The entry should go in the existing file in docs/content/c++/concepts/classes/classes.md

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Do the following:
1) Add a note (2-3 sentences) to the end of the entry about overloading.
2) Link to the overloading entry: https://www.codecademy.com/resources/docs/cpp/overloading

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by krushnarout**
### Description

Added notes about overloading and one example

Closes #3398 

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/classes/classes.md [hunk 1]
```diff
@@ -60,3 +60,54 @@ class City {
     bool is_capital;
 };
 ```
+
+## Overloading
+
+The method of creating operators that work differently with custom data types is called operator [overloading](https://www.codecademy.com/resources/docs/cpp/overloading) in C++. This enables the user to use these operators with objects of their own classes.
```
Comment (caupolicandiaz) [inline content/cpp/concepts/classes/classes.md:66]:
```suggestion
Redefining operators to work differently with custom data types is called operator [overloading](https://www.codecademy.com/resources/docs/cpp/overloading) in C++. This enables the user to use these operators with objects of their own classes.
```

File: content/cpp/concepts/classes/classes.md [hunk 2]
```diff
@@ -60,3 +60,54 @@ class City {
...
+
+```cpp
```
Comment (caupolicandiaz) [inline content/cpp/concepts/classes/classes.md:68]: update code to use 2 spaces for indentation instead of tabs

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@krushnarout Just added a couple of comments. Please update the entry, and tag me when you've completed the changes. Thanks

**Comment by krushnarout** [comment]
@caupolicandiaz I updated the notes with the suggested changes. Please review and let me know if any further changes are needed. Thanks!

**Review by caupolicandiaz — APPROVED** [review]
@krushnarout This is ready for a second review.

**Review by KTom101 — APPROVED** [review]
@krushnarout I added one change directly here: [3fe31cd](https://github.com/Codecademy/docs/pull/3476/commits/3fe31cd89edbe819353ae1fe28c89c4913af2575). Looks good to merge. 👍

**Comment by krushnarout** [comment]
@KTom101 Thank you

**Pull request merged**